src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/PanelMemberType.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import java.util.Arrays;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
```

Comment:
minor: I'm sure these enum constructors can be changed to `RequiredArgsConstructor` but if they're all like this then leave it for now.

Reply:
Yeah the other one was like this that I used as an example so will leave as is for now unless you want me to change it.

Reply:
That's fine.. just leave it as is then.